BetaCameras/RealSense2/RealSense2.NetStandard.csproj

```diff
@@ -47,6 +47,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Intel.RealSense">
+      <HintPath>Z:\external-libraries\Intel\libRealSense2\2.10.0-development\AnyCPU\netstandard2.0\Intel.RealSense.dll</HintPath>
```

Comment:
The binary is librealsense 2.10 plus the updated patches from [PR #1017](https://github.com/IntelRealSense/librealsense/pull/1017) that add netstandard2.0 compatibility.

Reply:
Alright. Let's see if they have plans to include that PR in a release soon.
Otherwise I'd try to create a fork which tracks the libRealSense development branch and pulls @sebandraos' patches.